### DISCUSSION TITLE
Lightning Layaway

### DISCUSSION THREAD

**Discussion by neurofighter** [btcpayserver/btcpayserver#2203] [Ideas / Feature Requests]
So I have 2 open lightning channels and would like to receive more bitcoin through strike but I don't have the inbound capacity.  If for example I wanted to buy a hardware wallet on Blockstream, but don't have enough send capacity in my lightning channels then it would be great if there was a layaway option.  I could make a first payment with all my send capacity, send more bitcoin to me cheaply via a service like strike, then make another payment to Blockstream.  This process could continue until the invoice was paid and then they could send me the device.

**Comment by neurofighter** [comment]
I think it would be great for the merchant!  They could set a time limit on the layaway (a couple of hours to a couple of days) and then if the total is not paid then they keep the deposit.  For me as a buyer it would be great because I could send smaller payments (larger likelihood of success) and lower my risk by not keeping as much locked up in a lightning channel.

**Reply by neurofighter** [reply]
I was also thinking about all the little random places I have bitcoin...exchange, mobile, lightning, cold storage, etc.  Rather than having to pay transaction fees to consolidate and then pay a transaction fee to make a payment, I could layaway and pay directly from each source

**Reply by neurofighter** [reply]
Just thinking that as purchases get larger on lightning you may have more parties wanting to participate in purchases.  I don't trust my broke cousin to borrow money and go to the grocery store but once I see he has paid 50% of his grocery bill I could pay the rest.

**Reply by pavlenex** [reply]
Did you try https://lnbig.com/ or [similar services? ](https://docs.btcpayserver.org/FAQ/FAQ-LightningNetwork/#how-can-i-get-inbound-capacity-to-my-node)

This is very hard to do in non-custodial software imo.

**Comment by Kukks** [comment]
This is already possible if a merchant wants such a workflow via payment requests.
